core/src/main/java/org/mule/runtime/core/privileged/exception/OnRuntimeProcessingStrategy.java

```diff
@@ -9,17 +9,24 @@
 import org.mule.runtime.api.component.location.ConfigurationComponentLocator;
 import org.mule.runtime.api.component.location.Location;
 import org.mule.runtime.core.api.construct.FlowConstruct;
+import org.mule.runtime.core.api.context.notification.FlowStackElement;
+import org.mule.runtime.core.api.event.CoreEvent;
 import org.mule.runtime.core.api.processor.ReactiveProcessor;
 import org.mule.runtime.core.api.processor.Sink;
 import org.mule.runtime.core.api.processor.strategy.ProcessingStrategy;
 import org.mule.runtime.core.privileged.processor.MessageProcessors;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 import reactor.core.publisher.Flux;
 import reactor.core.publisher.Mono;
 
 import java.util.Optional;
 
 public class OnRuntimeProcessingStrategy implements ProcessingStrategy {
```

Comment:
This should have been present in the previous review:

add a javadoc for this, maybe like this.
/**
 * A {@link ProcessingStrategy} used for {@link org.mule.runtime.core.internal.exception.GlobalErrorHandler}.
 * The resolution of the processing strategy is done in runtime depending on the flow location where the
 * error that is being handled was thrown.
 */